spec/routing/bulletins_routing_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe 'routes for bulletins', type: :routing do
+  describe 'GET /api/v1/bulletins/:id' do
+    let(:id) { 1 }
+    let(:show_bulletin) { get "/api/v1/bulletins/#{id}" }
+
+    it 'routes to bulletins#show(id)' do
+      expect(show_bulletin).to route_to(controller: 'api/v1/bulletins',
+                                        action: 'show',
+                                        id: '1',
+                                        format: 'json')
+    end
+  end
+
+  describe 'GET /api/v1/sunday' do
+    let(:show_sunday_bulletin) { get '/api/v1/sunday' }
+
+    it 'routes to bulletins#sunday' do
+      expect(show_sunday_bulletin).
+        to route_to(controller: 'api/v1/bulletins',
+                    action: 'sunday',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.